lib/Parser/Parse.cpp

```diff
@@ -2435,12 +2435,37 @@ bool Parser::IsImportOrExportStatementValidHere()
         && this->m_tryCatchOrFinallyDepth == 0;
 }
 
+template<bool buildAST> ParseNodePtr Parser::ParseImportCall()
+{
+    m_pscan->Scan();
+    ParseNodePtr specifier = ParseExpr<buildAST>(koplCma, nullptr, /* fAllowIn */FALSE, /* fAllowEllipsis */FALSE);
+    if (m_token.tk != tkRParen)
+    {
+        Error(ERRsyntax);
```

Comment:
Changing to 'ERRnoRparen'.

---
In reply to: [115324907](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r115324907) [](ancestors = 115324907)